sort.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const args = process.argv.slice(2);
+
+/*
+Usage:
+node sort.js {arguments}
+`arguments` is one or more file paths, separated by spaces.
+
+The script updates each file by sorting the labels by their keys.
+
+Example:
+node sort.js languages/de-DE/assess-api.json languages/de-DE/questions-api.json
+*/
+args.forEach((arg) => {
+    const text = fs.readFileSync(arg);
+    const parsedJson = JSON.parse(text);
+    const sortedKeys = Object.keys(parsedJson).sort();
+    let result = {};
+    
+    sortedKeys.forEach((key) => {
+        result[key] = parsedJson[key];
```

Comment:
Are we relying on a documented expectation that keys will always be sorted by order of addition in JS? Or is this just an implementation detail that happens to do what we want at the moment, but without guarantee that it will always be the case.

Reply:
Before ES2015, it used to be an implementation detail that didn't have any guarantees. In ES2015, it's been formalised in the spec:
https://exploringjs.com/es6/ch_oop-besides-classes.html#_traversal-order-of-properties

Reply:
Cool :+1: